templates/webapi/admin/group/group_property_editor.html.ep

```diff
@@ -83,15 +83,21 @@
                 <div class="mb-3 row">
                     <label for="editor-build-version-sort" class="form-label col-sm-2 control-label" data-bs-toggle="tooltip" title="Sets how builds are sorted when groups of them are displayed together">How to sort builds for display</label>
                     <div class="col-sm-10" id="editor-build-version-sort">
-                        <input type="radio" id="editor-build-version-sort-yes" name="build_version_sort" value="1"
-                            %= 'checked' if $group->build_version_sort
+                        <input type="radio" id="editor-build-version-sort-name" name="build_version_sort" value="1"
+                            %= 'checked' if $group->build_version_sort == 1
                             >
-                        <label class="form-label" for="editor-build-version-sort-yes" data-bs-toggle="tooltip" title="Treat the build values as version numbers and sort based on those">Sort by build value (as a version)</label>
+                        <label class="form-label" for="editor-build-version-sort-name" data-bs-toggle="tooltip" title="Treat the build values as version numbers and sort based on those">Sort by build value (as a version)</label>
                         <br>
-                        <input type="radio" id="editor-build-version-sort-no" name="build_version_sort" value="0"
-                            %= 'checked' if !$group->build_version_sort
+                        <input type="radio" id="editor-build-version-sort-recent" name="build_version_sort" value="0"
+                            %= 'checked' if $group->build_version_sort == 0
                             >
-                        <label class="form-label" for="editor-build-version-sort-no" data-bs-toggle="tooltip" title="Sort builds by the time a job was most recently created for each build (choose this if the build values do not sort properly as version numbers)">Sort by time job most recently created</label>
+                        <label class="form-label" for="editor-build-version-sort-recent" data-bs-toggle="tooltip" title="Sort builds by the time a job was most recently created for each build (choose this if the build values do not sort properly as version numbers)">Sort by time job most recently created</label>
+
+                        <br>
+                        <input type="radio" id="editor-build-version-sort-oldest" name="build_version_sort" value="2"
+                            %= 'checked' if $group->build_version_sort == 2
+                            >
+                        <label class="form-label" for="editor-build-version-sort-oldest" data-bs-toggle="tooltip" title="Sort builds by the time the oldest job was created for each build (choose this if the build values do not sort properly as version numbers)">Sort by time job least recently created</label>
```

Comment:
I assume this third option hasn't actually been implemented yet. So I suggest you leave this change out for now (and maybe also change the range in the validation to 0 and 1 only). This should instead be part of the commit that actually implements this option.